plugin/src/main/java/io/fabric8/maven/plugin/ResourceMojo.java

```diff
@@ -236,6 +239,18 @@ public void executeInternal() throws MojoExecutionException, MojoFailureExceptio
     private void lateInit() {
         platformMode = clusterAccess.resolvePlatformMode(mode, log);
 
+        if (isOpenShiftMode()) {
+            Properties properties = project.getProperties();
+            if (!properties.contains(DOCKER_IMAGE_USER)) {
+                String namespace = clusterAccess.getNamespace();
+                log.info("Using docker image name of namespace: " + namespace);
+                properties.setProperty(DOCKER_IMAGE_USER, namespace);
+            }
+            if (!properties.contains(PlatformMode.FABRIC8_EFFECTIVE_PLATFORM_MODE)) {
```

Comment:
yes - FABRIC8_EFFECTIVE_PLATFORM_MODE is just a way to detect the effective platform across various utility functions and mojos. 

DOCKER_IMAGE_USER is used by docker maven plugin to default the group (that would need fixing in d-m-p)

I've gone with `fabric8.internal....` to its clear its an internal flag not something a user should use